vector/src/main/java/im/vector/app/features/home/room/list/home/HomeRoomListFragment.kt

```diff
@@ -170,9 +165,7 @@ class HomeRoomListFragment :
         concatAdapter.registerAdapterDataObserver(firstItemObserver)
     }
 
-    override fun invalidate() = withState(roomListViewModel) { state ->
-        views.stateView.state = state.state
-    }
+    override fun invalidate() = Unit
```

Comment:
do we need this?

Reply:
In this case I prefer to listen just what changes

Reply:
OK, will do later, this is not a big deal.